scripts/install

```diff
@@ -195,92 +214,78 @@ ynh_app_setting_set $app ip4_dns1 80.67.169.12
 ynh_app_setting_set $app ip4_nat_prefix 10.0.242
 ynh_app_setting_set $app vpnclient no
 ynh_app_setting_set $app service_name $service_name
+ynh_app_setting_set $app packages $packages
```

Comment:
Uuuuh, I'm not so convinced that we need to save those in the settings ? Any particular motivation to do so ?

Reply:
I can retrieve it from `$firmware_nonfree` so it's as you wish :-)